assets/pages/new-contributions-landing/contributionsLandingInit.js

```diff
@@ -57,12 +59,22 @@ function initialisePaymentMethods(state: State, dispatch: Function) {
   loadPayPalRecurring().then(() => dispatch(setPayPalHasLoaded()));
 }
 
+function initialiseSelectedAnnualAmount(state: State, dispatch: Function) {
+  const { countryGroupId } = state.common.internationalisation;
+  const annualTestVariant = state.common.abParticipations.annualContributionsRoundThree;
```

Comment:
this should be fine since we know that `annualContributionsRoundThree` is in the test definitions. but for safety let's check on the existence of this property first - in case some, say, removes the test definition and then causes initialisation code to break